packages/@aws-cdk/app-delivery/package.json

```diff
@@ -62,7 +62,7 @@
     "cdk"
   ],
   "peerDependencies": {
-    "@aws-cdk/cdk": "^0.15.2",
-    "@aws-cdk/aws-codepipeline-api": "^0.15.2"
+    "@aws-cdk/aws-codepipeline-api": "^0.15.2",
```

Comment:
The peer dependencies, they ain't updated, and for some reason, it feels wrong.

Reply:
It is wrong, too